ibm_catalog.json

```diff
@@ -0,0 +1,184 @@
+{
+    "products": [
+      {
+        "name": "deploy-arch-ibm-scc",
```

Comment:
This has to match what you put in .catalog-onboard-pipeline.yaml. And also the value you have now is already in use for SCC DA. Please update to `deploy-arch-ibm-workload-protection`